<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_ventes</class>
 <widget class="QDialog" name="gestion_ventes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_gere_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 background-color: rgb(204, 204, 204);;font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer les ventes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer_fournisseur">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>570</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Algerian&quot;;
 background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>SUPPRIMER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>540</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:20px;
 font: 20pt &quot;Segoe Script&quot;;border:2px solid black;
 color: rgb(12, 255, 125);</string>
   </property>
   <property name="text">
    <string>PHARMACIE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_fournisseur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les fournisseurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accueilgerat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);
 font-size:15px;

 border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajouter_fournisseur">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>280</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Algerian&quot;;
 background-color: rgb(58, 140, 14);</string>
   </property>
   <property name="text">
    <string>AJOUTER</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonProfil_gerant">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
 background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="nome_user">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
 font: 900 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:15px;
 font: 600 16pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Dr.</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_rupture_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/devenir-pharmacien-921x1024.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Modifier_fournisseur">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>420</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Algerian&quot;;
 background-color: rgb(255, 222, 58);</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/OIP (3).jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_utilisateurs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les utilisateurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_stock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer le stock</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>881</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;border:2px solid green;
 background-color:#81c1b3;
 color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>Recherche</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>Code médicament :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date_vente">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>Quantité :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Quantit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>19</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id_vente">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>Date de vente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_codeMed">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>59</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>Code Client :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Recherchevente">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
 background-color: rgb(70, 225, 135);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>RECHERCHER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>Identifiant de vente :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_idClient">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>59</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid green;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_user">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
 color:#000;
 border:0;</string>
    </property>
    <property name="text">
     <string>ID_user</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>-130</y>
      <width>961</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border:2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableventeWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>821</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
 ;border:1px solid  black ;border-radius:5px;
 background-color: rgb(225, 255, 255);
 color: rgb(0, 0, 0);
 font: 700 10pt &quot;Segoe UI&quot;;
 text-align:center;
 td{
         background-color: rgb(150, 152, 120)
 }</string>
   </property>
   <column>
    <property name="text">
     <string>ID_vente</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantite_vente</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date_vente</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID_Medicament</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id_user</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID_Client</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>115</red>
      <green>176</green>
      <blue>172</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;border-radius:4px;
 font: 14pt &quot;Rockwell&quot;;
 background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonquitter">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;color:white;border-radius:10px;
 font: 900 12pt &quot;Arial Black&quot;;
 background-color: rgb(255, 114, 116);</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/WhatsApp Image 2024-04-30 à 20.55.44_6b270705.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>pushButton_gerer_utilisateurs</zorder>
  <zorder>pushButton_gere_vente</zorder>
  <zorder>pushButton_supprimer_fournisseur</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_gerer_fournisseur</zorder>
  <zorder>pushButton_accueilgerat</zorder>
  <zorder>pushButton_Ajouter_fournisseur</zorder>
  <zorder>line_2</zorder>
  <zorder>ButtonProfil_gerant</zorder>
  <zorder>nome_user</zorder>
  <zorder>label_6</zorder>
  <zorder>line</zorder>
  <zorder>pushButton_gerer_rupture_2</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton_Modifier_fournisseur</zorder>
  <zorder>label_5</zorder>
  <zorder>line_4</zorder>
  <zorder>pushButton_gerer_stock</zorder>
  <zorder>groupBox</zorder>
  <zorder>tableventeWidget</zorder>
  <zorder>label_date</zorder>
  <zorder>label_15</zorder>
  <zorder>pushButtonquitter</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
